### PR TITLE
ci: update github actions

### PR DESCRIPTION
**Pull request by amareshsm**
**Description: **

Updated GitHub actions

### CODE REVIEW

**Comment by Ryuno-Ki** [comment]
Changelog: https://github.com/actions/checkout/releases and https://github.com/actions/setup-node/releases

Notably, it appears to run on Node 16 then.

**Comment by amareshsm** [comment]
Can we specify node-version as 16.x?

**Comment by Ryuno-Ki** [comment]
I'm pointing this out, since the matrix lists Node v14:

https://github.com/11ty/eleventy/blob/5a00a5946b868ee82f6a09590fb7d4c8625117ac/.github/workflows/ci.yml#L11

**Comment by amareshsm** [comment]
> I'm pointing this out, since the matrix lists Node v14:
> 
> https://github.com/11ty/eleventy/blob/5a00a5946b868ee82f6a09590fb7d4c8625117ac/.github/workflows/ci.yml#L11

Removed matrix lists and specified node version (`16.x`)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -11,11 +11,11 @@ jobs:
...
     name: Node.js ${{ matrix.node }} on ${{ matrix.os }}
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Setup node
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v3
         with:
-          node-version: ${{ matrix.node }}
+          node-version: '16.x'
```
[the remark this answers is not in the document]
Comment (amareshsm) [reply .github/workflows/ci.yml]: Reverted this change.

**Comment by zachleat** [comment]
LGTM thanks!

**Pull request merged**